code/game/gamemodes/changeling/changeling.dm

```diff
@@ -16,25 +16,14 @@ GLOBAL_LIST_INIT(possible_changeling_IDs, list("Alpha","Beta","Gamma","Delta","E
 	required_players = 15
 	required_enemies = 1
 	recommended_enemies = 4
-
-	var/const/prob_int_murder_target = 50 // intercept names the assassination target half the time
-	var/const/prob_right_murder_target_l = 25 // lower bound on probability of naming right assassination target
-	var/const/prob_right_murder_target_h = 50 // upper bound on probability of naimg the right assassination target
-
-	var/const/prob_int_item = 50 // intercept names the theft target half the time
-	var/const/prob_right_item_l = 25 // lower bound on probability of naming right theft target
-	var/const/prob_right_item_h = 50 // upper bound on probability of naming the right theft target
-
-	var/const/prob_int_sab_target = 50 // intercept names the sabotage target half the time
-	var/const/prob_right_sab_target_l = 25 // lower bound on probability of naming right sabotage target
-	var/const/prob_right_sab_target_h = 50 // upper bound on probability of naming right sabotage target
-
-	var/const/prob_right_killer_l = 25 //lower bound on probability of naming the right operative
-	var/const/prob_right_killer_h = 50 //upper bound on probability of naming the right operative
-	var/const/prob_right_objective_l = 25 //lower bound on probability of determining the objective correctly
-	var/const/prob_right_objective_h = 50 //upper bound on probability of determining the objective correctly
-
+	/// The total number of changelings allowed to be picked.
 	var/changeling_amount = 4
+	/// A list containing references to the minds of soon-to-be changelings. This is seperate to avoid duplicate entries in the `changelings` list.
+	var/list/datum/mind/pre_changelings = list()
+
+/datum/game_mode/changeling/Destroy(force, ...)
+	pre_changelings.Cut()
```

Comment:
Make sure they get removed from the pool if their mind ever gets destroyed from cryoing or whatever.

Reply:
They're cleared when they're picked as well: 

![image](https://user-images.githubusercontent.com/42044220/174454718-9fce50f6-4417-4570-b4c7-ca081f14994f.png)

Reply:
aight